### ISSUE TITLE
Problem with puppet_agent_custom_options

### ISSUE THREAD

**Issue by kpu32167** [diodonfrost/ansible-role-puppet-agent#3]
##### SUMMARY

When I try to use puppet_agent_custom_options I got something like this [u'certname = test.ansible3', u'daemonize = true']

##### STEPS TO REPRODUCE

```yaml
- hosts: init_puppet
  become: yes
  roles:
    - role: diodonfrost.puppet_agent
      vars:
        - company_tag: "test"
        - puppet_version: "6"
        - puppet_agent_server: puppet.some.domain
        - puppet_agent_custom_options:
            - certname = {{ company_tag }}.{{ ansible_hostname }}
            - daemonize = true
```

##### EXPECTED RESULTS

```paste below
#####################################
##                                 ##
## THIS FILE IS MANAGED BY ANSIBLE ##
##                                 ##
#####################################

[main]
server = puppet.some.domain
environment = production
runinterval = 30m

certname = test.ansible3
daemonize = true
```

##### ACTUAL RESULTS

```paste below
#####################################
##                                 ##
## THIS FILE IS MANAGED BY ANSIBLE ##
##                                 ##
#####################################

[main]
server = puppet.some.domain
environment = production
runinterval = 30m

[u'certname = test.ansible3', u'daemonize = true']
```

##### HOW I FIXED IT
Changed in puppet.conf.j2
{{ puppet_agent_custom_options }}  
to 
{% for custom_option in (puppet_agent_custom_options | from_yaml) %}                                    
{{ custom_option }}                                                                                     
{% endfor %}

**Comment by diodonfrost** [comment]
Thanks for the report, I'll have a look

**Comment by diodonfrost** [comment]
Fixed in version 1.6.0
